addons/studio-tools/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Studio manager tools",
-  "description": "Lets you easily promote or remove anyone from your studios, as well as leaving any studio, through the \"curators\" tab.",
+  "name": "Studio tools",
+  "description": "Adds tools to easily promote to a manager or remove anyone from your studios through the \"curators\" tab, as well as leaving any studio using a button next to \"Follow Studio\".",
```

Comment:
Since 3.0 studio is soon, we should make it future-proof. Leave Studio button is under the Follow button in the new version.

Reply:
That's what I meant by "next to Follow Studio" (in the current studios it's on the curators tab), but I should probably change it to say "under".